FrEee.Tests/post-build.bat

```diff
@@ -4,9 +4,9 @@ for /f "tokens=1,* delims= " %%a in ("%*") do set ALL_BUT_FIRST=%%b
 
 echo Copying assets to %ALL_BUT_FIRST%\bin\%1\net472

```

Comment:
Should this line be changed? There is no net472 folder anymore...

Reply:
I've changed it to reflect the new folder structure.